src/utils/Text.js

```diff
@@ -0,0 +1,12 @@
+export function buttonText(
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do. **Please, correct it everywhere in the code**


`setButtonText`